sycl/test/hier_par/hier_par_wgscope.cpp

```diff
@@ -6,279 +6,18 @@
 //
 //===----------------------------------------------------------------------===//
 
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple \
+// RUN:   -I%S/src %s -o %t.out
```

Comment:
I suggest keeping the old command. There is no need in additional include paths relative to the source file directory.

Reply:
ok